packages/service-library/src/servicelib/fastapi/prometheus_instrumentation.py

```diff
@@ -7,18 +7,20 @@
 
 def setup_prometheus_instrumentation(app: FastAPI) -> Instrumentator:
 
-    instrumentator = (
-        Instrumentator(
-            should_instrument_requests_inprogress=True, inprogress_labels=False
-        )
-        .instrument(app)
-        .expose(app, include_in_schema=False)
-    )
-
-    def _unregister():
-        for collector in list(instrumentator.registry._collector_to_names.keys()):
+    instrumentator = Instrumentator(
+        should_instrument_requests_inprogress=True, inprogress_labels=False
+    ).instrument(app)
+
+    async def on_startup(app: FastAPI) -> None:
```

Comment:
MIONR: prefix with `_`

Reply:
Can you please also add the link  to
![image](https://github.com/user-attachments/assets/b8d2f4b1-723f-44da-b1c5-bdf5efc89728)

Reply:
done